test-clients/src/main/resource/contract/solidity/TransferAliased.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity ^0.8.12;
+
+import './AliasedOperator.sol';
+
+contract AliasedTransfer {
+    address aliasedOperator;
+
+    constructor() public {}
+
+    function initialize(address token) external returns (address operator) {
+        bytes memory bytecode = type(AliasedOperator).creationCode;
+        bytes32 salt = keccak256(abi.encodePacked(token));
+        assembly {
+            operator := create2(0, add(bytecode, 32), mload(bytecode), salt)
+        }
+        AliasedOperator(operator).initialize(token);
+        aliasedOperator = operator;
+    }
+
+    function transfer(address to, uint value) public {
+        AliasedOperator(aliasedOperator).transfer(to, value);
+    }
+
+    function associateOperator() public {
+        AliasedOperator(aliasedOperator).associate();
+    }
+
+    function addLiquidity(address token, address sender, int64 value) public {
```

Comment:
This function name is misleading and I don't think that we need it at all.

Reply:
We need it in order to be able to execute a transfer. If there is insufficient balance of the particular token, the transfer fails.